### PR TITLE
Fixed UI issues in PB > Users > Roles

### PR DESCRIPTION
**Pull request by mean2me**
PersonaBar > Users > Roles was not displayed due to an error with debounce library. I couldn't make it work, so I choosed to remove it and adjust the code accordingly.

Build settings cleanup
CSS fixes
Code refactoring

### CODE REVIEW

**Review by valadas — APPROVED** [review]
Looks good to me and tested too. Can we get a second review please @dnnsoftware/ae-approvers

**Review by daguiler — APPROVED** [review]

**Pull request merged**
